udata/forms/fields.py

```diff
@@ -285,6 +284,14 @@ def iter_choices(self):
             yield (value, label, selected)
 
 
+def slug(value):
+    return slugify(value.lower())
+
+
+def tags_list(value):
+    return list(set(slug(tag) for tag in value.split(',') if tag.strip()))
+
+
```

Comment:
I'd prefer to have it in a separate module because it's not more related to forms than harvest.
I don't like `harvest` having a new dependency on `udata.forms.fields` :/